src/deprecation.js

```diff
@@ -5,6 +5,26 @@ var core = require('./core'),
     extras = require('./extras'),
     filters = require('./filters');
 
+// provide method to give a stack track for warnings
+// useful for tracking-down where deprecated methods/properties/classes
+// are being used within the code
+function warn(msg) {
+    var stack = new Error().stack; 
+
+    // chop off the stack trace which includes pixi.js internal calls
+    stack = stack.split('\n').splice(3).join('\n');
```

Comment:
Is it guaranteed to only be 3 stack levels? I feel like this might be too fragile. Maybe detect if a line is internal to pixi and filter it instead?